### DISCUSSION TITLE
Writing parameter fields (Kc, FC and porosity) in a file

### DISCUSSION THREAD

**Discussion by cuneyd** [mhm-ufz/mHM#40] [Q&A] [answered]
Dear @lese66  and @rohinikumar82 ,

My question is about Figure 9 of your WRR-2010 paper below.

We wonder if there is an option to write out parameter fields (maps) of Kc or FC or porosity like in Figure 9 in the WRR paper below. In some models like Mike-She, there is an option for the user to plot these fields. 
I believe that we can always dig the code to save these fields in a file. 

Samaniego L., R. Kumar, S. Attinger (2010): Multiscale parameter regionalization of a grid-based hydrologic model at the mesoscale. Water Resour. Res., 46,W05523, doi:10.1029/2008WR007327. WRR Editors' Choice Award 2010

Thanks in advance,

Kind Regards,
Cüneyd

**Comment by mhm-admin** [comment, marked as answer]
Hi @cuneyd ,

of course you can see them at resolutions L1. Just crate a restart file at the point you want to see the parameter field and then look with ncview the file mHM_restart_00x.nc There in, you will find everything. If you want to see the parameter at another resolution, chance the L1 grid size. Thats all.

Good luck.

Luis

**Reply by mhm-admin** [reply]
For example sat theta is  L1_soilMoistSat, this need to be divided by the depth of the layer to get in the range [0-1], dimension less

**Reply by cuneyd** [reply]
Dear @lese66,

Thank you very much for the quick and nice tip.
write_restart = .true. is the key. 

Cheers,
Cüneyd
